cmd/controller-manager/main.go

```diff
@@ -90,6 +93,8 @@ func main() {
 
 	// parse allowed resources string
 	allowedGroupsSet := util.ParseAPIGroupsString(allowedGroupsString)
+	// parse controllers string
+	ctlrsToStart := util.ParseControllersString(controllers)
```

Comment:
I think that ParseControllersString belongs in this package rather than util, because ParseControllersString is not of general interest.

Reply:
Just wandering, is it common seen that there are *_test.go files in package main? Asking because I wrote a unit test for this func.

Reply:
I don't know about common, but it is certainly acceptable.

Reply:
Moved code from util to main.